crates/runtime/src/datafusion.rs

```diff
@@ -284,6 +293,30 @@ impl DataFusion {
         Ok(())
     }
 
+    pub async fn trigger_table_refresh(&self, dataset_name: &str) -> Result<()> {
+        let table = self
+            .ctx
+            .table_provider(OwnedTableReference::bare(dataset_name.to_string()))
```

Comment:
Seems like there should be some output to console like `Refresh for dataset <name> started...`

Reply:
Confirm there is already a system message when refresh is initiated 👍
```bash
2024-04-12T02:34:01.868025Z  INFO runtime::accelerated_table: Refreshing data for recent_blocks
```